var/spack/repos/builtin/packages/neuron/package.py

```diff
@@ -44,26 +44,31 @@ class Neuron(Package):
     version('7.4', '2c0bbee8a9e55d60fa26336f4ab7acbf')
     version('7.3', '993e539cb8bf102ca52e9fefd644ab61')
     version('7.2', '5486709b6366add932e3a6d141c4f7ad')
-    version('develop', git=github)
+    version('develop', git=github, preferred=True)
 
-    variant('mpi',           default=True,  description='Enable MPI parallelism')
-    variant('python',        default=True,  description='Enable python')
-    variant('shared',        default=False, description='Build shared libraries')
+    variant('binary',        default=True, description="Create special as a binary instead of shell script")
+    variant('coreneuron',    default=True, description="Patch hh.mod for CoreNEURON compatibility")
     variant('cross-compile', default=False, description='Build for cross-compile environment')
+    variant('debug',         default=False, description='Build debug with O0')
+    variant('knl',           default=False, description="Enable KNL specific flags")
+    variant('mpi',           default=True,  description='Enable MPI parallelism')
     variant('multisend',     default=True,  description="Enable multi-send spike exchange")
+    variant('profile',       default=False, description="Enable Tau profiling")
+    variant('python',        default=True,  description='Enable python')
+    variant('shared',        default=True, description='Build shared libraries')
     variant('rx3d',          default=False, description="Enable cython translated 3-d rxd")
 
-    depends_on('flex',       type='build')
-    depends_on('bison',      type='build')
-    depends_on('automake',   type='build')
-    depends_on('automake',   type='build')
     depends_on('autoconf',   type='build')
+    depends_on('automake',   type='build')
+    depends_on('bison',      type='build')
+    depends_on('flex',       type='build')
     depends_on('libtool',    type='build')
     depends_on('pkg-config', type='build')
 
     depends_on('mpi',         when='+mpi')
-    depends_on('python@2.6:', when='+python')
     depends_on('ncurses',     when='~cross-compile')
+    depends_on('python@2.6:', when='+python')
```

Comment:
I would really not base any packages on python 2.6 - unless there is a very specific need to support this version - just require 2.7+

Reply:
We put minimum version that package can run with (which is 2.6).
But, as python recepie has latest 3.6.xx as version, Spack will always use latest python to build. So this is not a problem.

By the way, the 2.6 was default from BG-Q ;)